### DISCUSSION TITLE
Problem with TTPLANET_Controlnet_Tile_realistic_v2

### DISCUSSION THREAD

**Discussion by Yakonrus** [huggingface/diffusers#7723] [Q&A] [answered]
https://huggingface.co/TTPlanet/TTPLanet_SDXL_Controlnet_Tile_Realistic/discussions/2
Does anyone know if it is possible to use it in ControlNetModel?

model = ControlNetModel.from_single_file(
    './TTPLANET_Controlnet_Tile_realistic_v2_fp16.safetensors',
    config_file='./config.json',
)

"cond_stage_config" in original_config["model"]["params"]
KeyError: 'model'

**Comment by tolgacangoz** [comment, marked as answer]
Could you rename `TTPLANET_Controlnet_Tile_realistic_v2_fp16.safetensors` as `diffusion_pytorch_model.safetensors` and put it in a folder with the config file, then run:
```py
model = ControlNetModel.from_pretrained("<folder_name>")
```

**Comment by Yakonrus** [comment]
I'm a little late :)

In general, I uploaded the renamed file diffusion_pytorch_model.fp16.safetensors.
https://huggingface.co/Yakonrus/SDXL_Controlnet_Tile_Realistic_v2
It's working
🤷‍♂️
I think there is a bug in loading from files with names not like in diffusers
